drivers/fpga/Kconfig

```diff
@@ -10,7 +10,7 @@ menuconfig FPGA
 
 if FPGA
 
-module = fpga
+module = FPGA
```

Comment:
This change is needed so it will be `CONFIG_FPGA_LOG_LEVEL` and not `CONFIG_fpga_LOG_LEVEL`